perllib/FixMyStreet/Geocode/OSPlaces.pm

```diff
@@ -0,0 +1,24 @@
+=head1 NAME
+
+FixMyStreet:Geocode::OSPlaces - OS Places API reverse geocoding for FixMyStreet
+
+=cut
+
+package FixMyStreet::Geocode::OSPlaces;
+
+use strict;
+use warnings;
```

Comment:
Think this needs a `use FixMyStreet::Geocode;` or L18 below crashes when called during `send-reports`.